# This srv file defines the service
# for a need to execute a given trajectory (traj + armType)

uniform_object_rearrangement/ArmTrajectory arm_trajectory
---
bool success